src/lib/blueprint.ts

```diff
@@ -49,6 +55,15 @@ interface Request {
 
 interface Response {
   description: string
+  responseType: 'resource' | 'resource_list' | 'void'
+  responseKey: string | null
+  resourceType: string | null
+}
+
+interface Property {
+  name: string
+  type: 'string' | 'enum' | 'record' | 'list' | 'object'
+  properties: Property[] | null
 }
```

Comment:
We may need to do something similar for this, e.g.,

```	
interface BaseProperty {
  name: string
  type: 'string' | 'enum' | 'record' | 'list' | 'object'
  properties: Property[] | null
}

interface ObjectProperty extends BaseProperty {
  type: 'object'
  properties: Property[]
}

interface EnumProperty extends BaseProperty {
  type: 'object'
  values: string[]
}
...
```

Reply:
@kainpets Actually, it might be worth introducing the EnumValue type since giving us an extension point to add extra metadata for each enum value may be useful. I don't think we need to anticipate too much complexity like this, but this case seems reasonable.